model/pipeline/src/main/proto/schema.proto

```diff
@@ -37,6 +37,8 @@ message Schema {
   // REQUIRED. An RFC 4122 UUID.
   string id = 2;
   repeated Option options = 3;
+  // Indicates that encoding positions have been overridden.
+  bool encoding_positions_set = 4;
```

Comment:
Why do we need to add this to the proto? Can't we continue to rely on checking if encoding_position is set on the fields?

Reply:
Ack thank you.

nit: can you update the comment for encoding_position comment to reflect this, something like:

```
   // OPTIONAL. The position of this field's data when encoded, e.g. with beam:coder:row:v1.
   // Used to support backwards compatibility with schema changes.
   // If this Field is part of a Schema where encoding_positions_set is True then encoding_position must be 
   // defined, otherwise this field is ignored.
   int32 encoding_position = 5;
 ```

Reply:
I meant the comment Field.encoding_position